### ISSUE TITLE
Regression: hovering icons on tasks cards on the board does not highlight/opacify them

### ISSUE THREAD

**Issue by nekohayo** [kanboard/kanboard#4214]
Running the latest kanboard version, I noticed that for tasks on a board, in the icons row for things like "Relations", "Subtasks", "Comments", "Description", if you hover the mouse over these semi-transparent icons they don't "highlight" (become fully opaque) like they used to before, to indicate that the mouse is correctly over the target, _except_ in the case of the "comments" icon.

I found the opacification on mouseover very useful to know that my mouse is correctly positioned with precision over the target, before the expanded popup preview even shows up.

So, to summarize, on mouseover:
* Relationships icon don't opacify
* Subtasks icon doesn't opacify
* Comments icon DOES opacify, which is nice
* Description icon doesn't opacify

**Comment by rafacamargo123** [comment]
MR #4217 opened.
New behaviour:
![ezgif-3-5a9f5ebb4806](https://user-images.githubusercontent.com/12586009/59734679-cac89180-9228-11e9-8e10-37996942992e.gif)
